opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQEnums.hpp

```diff
@@ -76,6 +76,7 @@ enum class UDQTokenType{
     number = 1,
     open_paren = 2,
     close_paren = 3,
+    comp_expr = 6,
```

Comment:
This does not seem to be used.

Reply:
This item comp_expr - is used in AggregateUDQData.cpp